aws-serverless-java-container-springboot3/pom.xml

```diff
@@ -17,7 +17,7 @@
     <properties>
         <spring.version>6.0.9</spring.version>
         <springboot.version>3.1.0</springboot.version>
-        <springsecurity.version>6.1.0</springsecurity.version>
+        <springsecurity.version>6.0.3</springsecurity.version>
```

Comment:
this downgrade seems to be a merge error

Reply:
Yes. Thanks for pointing that out.